### PR TITLE
Bugfix part2

### PR DESCRIPTION
**Pull request by silentswordxv**
# Description

Implements/fixes [SOEN-156](https://soen390team8.atlassian.net/browse/SOEN-156?atlOrigin=eyJpIjoiNzlhZThkYzRkZDFlNDA3YThhY2I5YzM1ODQ2M2M2NWUiLCJwIjoiaiJ9)

- Changed custom function to built-in

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

manual check: Sign Out Redirect working on all pages for all users

### CODE REVIEW

**Review by stevenvuong00 — APPROVED** [review]
Good catch!

**Review by SergioAAI — APPROVED** [review]
Good job!!

File: frontend/components/login-logout-button.tsx [hunk 1]
```diff
@@ -1,24 +1,17 @@
...
-    }
 
     return (
         <Button
             backgroundColor={MAIN_COLOR}
             _hover={{ opacity: "90%" }}
             size="sm"
             color="white"
-            onClick={() => (session ? signOutRedirect() : signIn())}>
+            onClick={() => (session ? signOut({callbackUrl: "/"}) : signIn())}>
```
Comment (NicholasYip) [inline frontend/components/login-logout-button.tsx:14]: very smart stuff!

**Review by NicholasYip — APPROVED** [review]
High quality work as always!

**Pull request merged**